src/components/MyProfilePage/Profile.tsx

```diff
@@ -79,7 +79,9 @@ export default function Profile() {
       <S.PrivacySection>
         <h1>{user.name}</h1>
         <p>
-          {user.grade + '학년 ' + user.classNum + '반 ' + user.number + '번'}
+          {!user.grade && !user.classNum && !user.number
+                ? '졸업생'
+                : `${user.grade}학년 ${user.classNum}반 ${user.number}`}
```

Comment:
번호가 빠진거같아 suggestion 드려요!
```suggestion
                : `${user.grade}학년 ${user.classNum}반 ${user.number}번`}
```

Reply:
악 커밋 빠졌네요

Reply:
@baekteun 수정 완료했습니다